analysis_scripts/analyze_quote_data/location-analysis_all_years.md

```diff
@@ -0,0 +1,234 @@
+    ## file to create a valid file.
+
+``` r
+full_loc_df = full_loc_df[-1,]
+
+full_loc_df = subset(full_loc_df, est_un_region != "" & 
+                                        est_un_subregion != "" &
+                                        est_un_region != "NO_EST" & 
+                                        est_un_subregion != "NO_EST")
+
+head(full_loc_df)
+```
+
+    ##   est_country_code            file_id                              text
+    ## 2               ae news.2010.647.html rochester institute of technology
+    ## 3               ae news.2010.491.html rochester institute of technology
+    ## 4               ae       468609a.html                               ias
+    ## 5               af       464014b.html                       afghanistan
+    ## 6               af news.2010.684.html                       afghanistan
+    ## 7               af       465990a.html                       afghanistan
+    ##            ner          est_country est_un_region est_un_subregion year
+    ## 2 ORGANIZATION United Arab Emirates          Asia     Western Asia 2010
+    ## 3 ORGANIZATION United Arab Emirates          Asia     Western Asia 2010
+    ## 4 ORGANIZATION United Arab Emirates          Asia     Western Asia 2010
+    ## 5      COUNTRY          Afghanistan          Asia    Southern Asia 2010
+    ## 6      COUNTRY          Afghanistan          Asia    Southern Asia 2010
+    ## 7      COUNTRY          Afghanistan          Asia    Southern Asia 2010
+
+Now we join the tables together for comparison.
+
+``` r
+full_loc_df$is_benchmark = "full_data"
+bm_loc_df$is_benchmark = "benchmark"
+
+bm_full_loc_df = rbind(subset(full_loc_df, year %in% c("2010", "2015", "2020")), 
+                       bm_loc_df)
+bm_full_loc_df = subset(bm_full_loc_df, est_un_region != "" & 
+                                        est_un_subregion != "" &
+                                        est_un_region != "NO_EST" & 
+                                        est_un_subregion != "NO_EST")
+```
+
+### compare benchmark and non-benchmark data
+
+Now lets check how well the benchmark proportions match the full dataset proportions. The number of articles per year in each benchmark set is 10, the total number of articles in the year is between 400-1000. Since the number of articles in the benchmark set is very small, we don't expect exact consistency between the benchmark and total data sets.
+
+#### Lets look at the largest region definition first: UN region
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-4-1.png" style="display: block; margin: auto;" />
+
+#### Lets look at the UN Subregion level
+
+<img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-5-1.png" style="display: block; margin: auto;" /><img src="location-analysis_all_years_files/figure-markdown_github/unnamed-chunk-5-2.png" style="display: block; margin: auto;" />
```

Comment:
What's the conclusion from these plots? I see some differences, but as you said, the benchmark data is very small. I guess for your this is good enough, but I'm curious to know how you determined that.

Reply:
I think it's still hard to say exactly if the changes in frequencies are bugs or if the benchmark dataset was just unlucky. I think I need to look at it more, because these could be caused by inaccurate search results in the openstreetmap cache.
The main thing i was looking for was if there are large changes (>20%) in frequency between benchmark and the full dataset, and of these the ones I have to still look at are IT, GB, and CA. 
I added an issue to the repo to make sure this is looked at in greater detail: https://github.com/greenelab/nature_news_disparities/issues/8.

Reply:
I think that makes sense.